wpimath/src/test/native/cpp/system/NumericalIntegrationTest.cpp

```diff
@@ -53,6 +53,7 @@ TEST(NumericalIntegrationTest, RK4TimeVarying) {
 }
 
 // Tests that integrating dx/dt = 0 works with RKDP
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
This doesn't do what you'd think. It actually suppresses the warning for the rest of the file. It also will warn on compilers that aren't GCC. Use the `WPI_IGNORE_DEPRECATED` and `WPI_UNIGNORE_DEPRECATED` macros from `<wpi/deprecated.h>` instead.